.travis.yml

```diff
@@ -144,6 +144,66 @@ matrix:
       name: "CMake blinky example - debug"
       env: NAME=cmake_test EXAMPLE_NAME=mbed-os-example-blinky TARGET_NAME=K64F PROFILE=debug
 
+    - &cmake-build-test
+      stage: "CMake"
+      name: "CMake blinky example"
+      env: NAME=cmake_test EXAMPLE_NAME=mbed-os-example-blinky TARGET_NAME=K64F
+      language: python
+      python: 3.8
+      install:
+        # Install gcc
+        - source_pkg gcc
+        - arm-none-eabi-gcc --version
+        - pip install --upgrade cmake
+        - pip install --upgrade mbed-tools
+        # We use manual clone, with depth and single branch = the fastest
+        # Because of this, we need to create .mbed file as the current tools require it
+        - >-
+         git clone --depth=1 --single-branch --branch feature-cmake https://github.com/ARMmbed/${EXAMPLE_NAME}.git;
+         cd ${EXAMPLE_NAME};
+         git clone --depth=1 --single-branch https://github.com/ARMmbed/mbed-os.git;
+         cd mbed-os;
+         git fetch origin +refs/pull/${TRAVIS_PULL_REQUEST}/merge;
+         git checkout -qf FETCH_HEAD
+        - >-
+          cd -;
+          touch .mbed;
+          mbedtools configure -t GCC_ARM -m ${TARGET_NAME};
+          mkdir -p build
+      script:
+        - cd build && cmake .. -GNinja -DMBED_PROFILE=develop && cmake --build .
+
+    - <<: *cmake-build-test
+      name: "CMake kvstore example"
+      env: NAME=cmake_test EXAMPLE_NAME=mbed-os-example-kvstore TARGET_NAME=K64F
+
+    - <<: *cmake-build-test
+      name: "CMake ble example"
+      env: NAME=cmake_test EXAMPLE_NAME=mbed-os-example-ble TARGET_NAME=NRF52840_DK
```

Comment:
Only one example (BLE_Button) has CMake support. Will it work with this line or do you have to specify which sub-project to build?